smoke-test/tests/consistency_utils.py

```diff
@@ -1,8 +1,15 @@
 import time
+import os
 import subprocess
+
 _ELASTIC_BUFFER_WRITES_TIME_IN_SEC: int = 1
+USE_STATIC_SLEEP: bool = bool(os.getenv("USE_STATIC_SLEEP", False))
+ELASTICSEARCH_REFRESH_INTERVAL_SECONDS: int = int(os.getenv("ELASTICSEARCH_REFRESH_INTERVAL_SECONDS", 5))
 
 def wait_for_writes_to_sync(max_timeout_in_sec: int = 120) -> None:
+    if USE_STATIC_SLEEP:
+        time.sleep(ELASTICSEARCH_REFRESH_INTERVAL_SECONDS)
```

Comment:
Why not have this in Env variable then you can control the sleep seconds from eng var and you can still check if this variable exists to enable or disable it

Reply:
It is in env variable
```
ELASTICSEARCH_REFRESH_INTERVAL_SECONDS: int = int(os.getenv("ELASTICSEARCH_REFRESH_INTERVAL_SECONDS", 5))
```